apps/studio/src/background/lib/electron/ProtocolBuilder.ts

```diff
@@ -80,7 +80,7 @@ export const ProtocolBuilder = {
       const normalized = path.normalize(pathName)
       const fullPath = path.join(platformInfo.userDirectory, "plugins", normalized)
       log.debug("resolving", pathName, 'to', fullPath)
-      if (bksConfig.get(`plugins.${pluginId}.enabled`) === false) {
+      if (bksConfig.get(`plugins.${pluginId}.disabled`)) {
```

Comment:
lgtm. Would probably be good to have an api like `bksConfig.get('plugins', pluginId, 'disabled')` -- that takes any number of strings and joins them itself